fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -138,7 +138,7 @@
 
     protected FedoraResource resource;
 
-    private static long MAX_BUFFER_SIZE = 10240000;
+    private static long max_BUFFER_SIZE = 10240000;
```

Comment:
I suspect this issue could also be resolved by making `MAX_BUFFER_SIZE` `final`.